src/ng/filter/orderBy.js

```diff
@@ -176,88 +176,110 @@
 orderByFilter.$inject = ['$parse'];
 function orderByFilter($parse) {
   return function(array, sortPredicate, reverseOrder) {
+
     if (!(isArrayLike(array))) return array;
-    sortPredicate = isArray(sortPredicate) ? sortPredicate : [sortPredicate];
+
+    if (!isArray(sortPredicate)) { sortPredicate = [sortPredicate]; }
     if (sortPredicate.length === 0) { sortPredicate = ['+']; }
```

Comment:
Mmm... thinking a little more about this, I think my proposal and the current code are wrong as we are modifying the original `sortPredicate` and we should not do that